Gemfile

```diff
@@ -6,12 +6,24 @@ gem 'oat', '~> 0.4.5'
 
 gem 'pg'
 
+gem 'doorkeeper'
+
 gem 'bcrypt-ruby', '~> 3.1.5'
 
 gem 'pundit'
 
+# Server recommended for run apps in Heroku is Puma
+# Also they recomment manage app start with foreman
+gem 'foreman'
+gem 'puma'
+
 gem 'public_pages', path: "engines/public_pages"
 
+group :assets do
+  gem 'sass-rails'
+  gem 'uglifier'
+end
```

Comment:
Deploying to Heroku asset pipeline complained for not have this. Also I removed boostrap sourcemap file